tests/test_broadcasting.py

```diff
@@ -41,14 +41,14 @@
 
     kernels.Linear,
     kernels.Polynomial,
-    kernels.ArcCosine,
-    kernels.Periodic,
+    # kernels.ArcCosine,
+    # kernels.Periodic,
```

Comment:
They don't broadcast ?

Reply:
Not yet, I can have look if I can make them broadcast-able as well.